### PR TITLE
Update component boilerplate

### PR DESCRIPTION
**Pull request by simu**
* Introduces GitHub action which renders GitHub release when a tag is pushed

## Checklist

- [x] keep pull requests small so they can be easily reviewed.

### CODE REVIEW

**Comment by simu** [comment]
@ccremer does the configuration in this PR (copied from the component template in Commodore) respect the `ignore` label for the changelog generation?

**Comment by ccremer** [comment]
> @ccremer does the configuration in this PR (copied from the component template in Commodore) respect the `ignore` label for the changelog generation?

The changelog generator *only* considers the labels defined in the config. Unlabelled or unrecognized PRs are ignored. The `ignore` label really just makes it explicit (i.e. it was a deliberate decision to not include in the changelog).

File: Makefile [hunk 1]
```diff
@@ -5,28 +5,50 @@ SHELL := bash
 .DELETE_ON_ERROR:
 .SUFFIXES:
 
-FILES_JSONNET ?= $(shell find . -type f -name '*.*jsonnet' -or -name '*.libsonnet')
-FILES_YAML ?= $(shell find . -type f -name '*.yaml' -or -name '*.yml')
+DOCKER_CMD   ?= docker
+DOCKER_ARGS  ?= run --rm --user "$$(id -u)" -v "$${PWD}:/component" --workdir /component
```
[the remark this answers is not in the document]
Comment (simu) [reply Makefile:9]: Seems like something to fix in the component template, and then update in all components (hopefully we'll be able to implement an automated solution for that).

It probably won't work for local compiles the way it is currently, but I'm not going to spend time implementing compile tests for this component right now.

File: Makefile [hunk 2]
```diff
@@ -5,28 +5,50 @@ SHELL := bash
...
+YAMLLINT_DOCKER ?= $(DOCKER_CMD) $(DOCKER_ARGS) $(YAMLLINT_IMAGE)
+
+VALE_CMD  ?= $(DOCKER_CMD) $(DOCKER_ARGS) --volume "$${PWD}"/docs/modules:/pages vshn/vale:2.1.1
+VALE_ARGS ?= --minAlertLevel=error --config=/pages/ROOT/pages/.vale.ini /pages
+
+ANTORA_PREVIEW_CMD ?= $(DOCKER_CMD) run --rm --publish 2020:2020 --volume "${PWD}":/antora vshn/antora-preview:2.3.3 --style=syn --antora=docs
 
 .PHONY: all
-all: lint
+all: lint open
```
[the remark this answers is not in the document]
Comment (simu) [reply Makefile]: removed

**Review by ccremer — APPROVED** [review]

**Pull request merged**